### PR TITLE
Fix 12653 - Add simplest mode handler for -zzz flag ##bin

### PR DESCRIPTION
**Pull request by AdamVanScyoc**

### CODE REVIEW

**Comment by radare** [comment]
ping

File: libr/bin/bfile.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* radare2 - LGPL - Copyright 2009-2018 - pancake, nibble, dso */
 
+#include <r_cons.h>
```
Comment (radare) [inline libr/bin/bfile.c]: nope. you cant call rcons from rbin

File: libr/bin/bfile.c [hunk 2]
```diff
@@ -34,9 +35,21 @@ static void print_string(RBinFile *bf, RBinString *string) {
 	type_string = r_bin_string_type (string->type);
 	vaddr = addr = r_bin_get_vaddr (bin, string->paddr, string->vaddr);
 
+	// If raw string dump mode, use printf to dump directly to stdout.
+	PrintfCallback temp = io->cb_printf;
+	if (raw == 2) {
+		io->cb_printf = r_cons_printf;
```
Comment (radare) [inline libr/bin/bfile.c]: you can only do this from core, rcons is not visible by rbin

Comment (AdamVanScyoc) [reply libr/bin/bfile.c]: Previously you told me to use r_cons_printf (and not printf)

https://github.com/radare/radare2/pull/12578#discussion_r244532220

I will try to find another print function

**Comment by radare** [comment]
hello?

**Comment by radare** [comment]
Rconsprintf is available via io->cb_printf

**Pull request merged**